packages/mds-db/schema.ts

```diff
@@ -106,6 +109,9 @@ const TABLE_COLUMNS: { [T in TABLE_NAME]: Readonly<COLUMN_NAME[]> } = {
     COLUMN.audit_event_type,
     COLUMN.audit_issue_code,
     COLUMN.audit_subject_id,
+    COLUMN.provider_event_id,
+    COLUMN.provider_event_type,
+    COLUMN.provider_event_type_reason,
```

Comment:
You are going to need to write a migration to add these columns to the `audit_events` table.

Reply:
See this commit for how to do the migration btw: https://github.com/CityOfLosAngeles/mds-core/commit/bf241bff772b1efb2803c89f5e6721d63ed52d9b

Reply:
Thanks for the example @janedotx - I added the db migration code, assuming I'll merge this PR after yours